src/store/comment.js

```diff
@@ -60,7 +60,15 @@ export default {
 		}
 	},
 	actions: {
-		listComments({ commit }, card) {
+		listComments({ commit, state }, card) {
+			card.limit = 5
+			card.offset = 0
```

Comment:
I'd prefer to have an object in the comments store, that stores the actual offset. That way the regular card object is not polluted with metadata.